src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -385,6 +385,12 @@ private function getToolbarButtons()
                 'icon' => 'add_circle_outline',
                 'class' => 'btn-outline-primary',
             ];
+
+            $toolbarButtons['list_v2'] = [
```

Comment:
@JevgenijVisockij I added these buttons like in the edition page so that it's easier to toggle from one version to the other It will help us testing